liberty-managed/src/main/java/org/jboss/arquillian/container/was/wlp_managed_8_5/WLPManagedContainer.java

```diff
@@ -1413,7 +1413,7 @@ private Throwable createException(ExMsg x, Throwable cause) {
            // Caused by: org.jboss.weld.exceptions.DefinitionException:
 
            while ((line = br.readLine()) != null) {
-               Pattern p = Pattern.compile("(Stack Dump = |Caused by: )([\\p{L}\\\\p{N}$\\.]+?):(.*)");
+               Pattern p = Pattern.compile("(Stack Dump = |Caused by: )([\\p{L}\\p{N}$\\.]+?):(.*)");
```

Comment:
I both like and don't like this :stuck_out_tongue: 

On the one hand, using the unicode character categories is neat, but on the other, I had to go and look up what `L` and `N` were and they're actually far wider than we need because Java identifiers are restricted to the US-ASCII letters.

It's also still missing the underscore character.

Reply:
On the other hand, it's now unlikely that it would catch anyone out as underscores in class names are pretty rare.

Reply:
Java identifiers are not restricted to ASCII chars.
I will add underscores.